Middleware examples

lakovicj opened the pull request:
Two middlewares added:

- PrintRequest
Simple middleware. Prints request on every http request.

- CheckAge
Middleware is attached to the GET /test route. It prints message on the screen depending on age parameter from request.

lakovicj committed:
Middleware examples

dejan-stokic commented on laravel-example/app/Http/Middleware/CheckAge.php:
```diff
@@ -0,0 +1,31 @@
+            echo "<br><hr> Woops, 'age' param should be >18!";
+        }
+    }
+}
```
Dobro je sve organizovano, obratio si paznju na neophodne provere sto je super. Par sitnih stvari, po zahtevu provera treba da ukljuci i one koji imaju 18 godna.
Poslednji kondicioni blok moze biti samo _else_, nije neophodna dodatna provera.

Kao i pre sto smo pricali bilo bi dobro vratiti informativniji response umesto _echo_ poruke. U ovoj situaciji mozemo vratiti 403 response sa istim porukama.
Ovde mozes pogledati koji sve response kodovi postoje: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

lakovicj committed:
JLLR17: Laravel middleware FIX

Echo in CheckAge middleware replaced with 403 status response. Age condition fixed.

lakovicj committed:
Merge branch 'main' into feature/middleware
